src/OpenTK.Windowing.Desktop/NativeWindowSettings.cs

```diff
@@ -170,5 +170,53 @@ public NativeWindowSettings()
         /// otherwise multisampling is used if available. The actual number of samples is the closest matching the given number that is supported.
         /// </remarks>
         public int NumberOfSamples { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating the number of stencil bits used for OpenGL context creation.
+        /// </summary>
+        /// <remarks>
+        /// Default value is 8.
+        /// </remarks>
+        public int? StencilBits { get; set; }
```

Comment:
Can we not initialize these to the default values instead of using nullables?

If we can't do that because our defaults aren't known ahead of time, can we specify where the defaults will come from in the documentation? `GLFW.GetVideoMode(monitor).StencilBits`;` ?

Reply:
My idea with making them nullable is to give the user the choice to make OpenTK not call these functions for whatever reason. But as it seems we are setting them anyways by default I think that yeah the documentation should probably tell you that it's going to be called anyways (for the color bits, stencil and depth don't get called if they are null).